src/plugins/fn.ajax/fn.ajax.js

```diff
@@ -14,6 +14,11 @@ function ajax (action, opt, done, before) {
 
   if (typeof window.FormData === 'undefined' || !(opt.body instanceof window.FormData)) {
     opt.headers['Content-Type'] = opt.headers['Content-Type'] || 'application/x-www-form-urlencoded';
+
+    // If it's a GET request, the serialized body should turn into query params
```

Comment:
This should be a bit after, after [this line](https://github.com/dariush-alipour/umbrella/blob/1631e8f59f0314a170c1114fac49475556d9f381/src/plugins/fn.ajax/fn.ajax.js#L30) since the body might be an object at this stage.

Reply:
I will change this on my end though, no worries